Realm/Tests/Swift/SwiftSchemaTests.swift

```diff
@@ -82,13 +82,13 @@ class SwiftRecursingSchemaTestObject : RLMObject {
 }
 
 class InitAppendsToArrayProperty : RLMObject {
-    @objc dynamic var propertyWithIllegalDefaultValue: RLMArray = {
+    @objc dynamic var propertyWithIllegalDefaultValue: RLMArray<SwiftIntObject> = {
```

Comment:
Are these changes required by this PR, or are they being made preemptively in advance of #5149?